include/psp2/ctrl.h

```diff
@@ -28,13 +28,17 @@ enum {
 /** Enumeration for the digital controller buttons. */
 enum {
 	SCE_CTRL_SELECT     = 0x000001,	//!< Select button.
+	SCE_CTRL_L3         = 0x000002,	//!< L3 button.
+	SCE_CTRL_R3         = 0x000004,	//!< R3 button.
 	SCE_CTRL_START      = 0x000008,	//!< Start button.
 	SCE_CTRL_UP         = 0x000010,	//!< Up D-Pad button.
 	SCE_CTRL_RIGHT      = 0x000020,	//!< Right D-Pad button.
 	SCE_CTRL_DOWN       = 0x000040,	//!< Down D-Pad button.
 	SCE_CTRL_LEFT       = 0x000080,	//!< Left D-Pad button.
 	SCE_CTRL_LTRIGGER   = 0x000100,	//!< Left trigger.
 	SCE_CTRL_RTRIGGER   = 0x000200,	//!< Right trigger.
+	SCE_CTRL_L1         = 0x000400,	//!< L1 button.
```

Comment:
> so when used with normal functions, SCE_CTRL_LTRIGGER is L1 on dualshock and L1 on ps vita; 

yes

> however when used with sceCtrlReadBufferPositiveExt2 SCE_CTRL_LTRIGGER is L2 on dualshock and does not appear at all on ps vita?

actually psvita not have l2/r2/l3/r3 buttons, so not appear :) it look like swap.

> I guess if I got this right, you should specify for _L1/_R1 that these are only used for the sceCtrlReadBufferPositiveExt2 function

well...... `LTRIGGER` is can confuse.
in many cases(like xbox, ds, etc), `LTRIGGER` is not mean `L1`, mean L2 button.
but we use expect `L1`

How about define all values ? like;
```
{
  ...
  SCE_CTRL_LTRIGGER = 0x000100,
  SCE_CTRL_L1 = 0x000400, // or SCE_CTRL_EXT_L1 = 0x000400,
  SCE_CTRL_L2 = 0x000100, // duplicate define but...
  SCE_CTRL_L3 = 0x000002,
...
}
```

or want you another enum set? like;

```
{
  SCE_CTRL_EXT_L1 = 0x000400,
  SCE_CTRL_EXT_L2 = 0x000100,
  SCE_CTRL_EXT_L3 = 0x000002,
}
```
but this is little strange, maybe we use both enums at same time, we need to split?

and another options, define L2 to 0x000400, then make compatible user method, like `sceCtrlPeekBufferPositiveExtend`..